iFixit/react-commerce
# FPP Switch to Strapi: Remove redirect and logic from `/products`

First PR that for https://github.com/iFixit/ifixit/issues/48697. Here we are just getting rid of the redirects and cleaning up `/products` logic that included `pro-tech-toolkit` and `manta-driver-kit-112-bit-driver-kit`

## QA

Make sure when we visit `/products/pro-tech-toolkit` and `/products/manta-driver-kit-112-bit-driver-kit` the page loads to the normal product pages.

CC: @sterlinghirsh
